release-calendar/src/client/components/App.tsx

```diff
@@ -23,31 +27,69 @@ import {getEvents} from '../models/release-event';
 
 interface AppState {
   events: EventSourceInput[];
+  selectedChannel: number;
 }
 
 export class App extends React.Component<{}, AppState> {
-  constructor(props: unknown) {
-    super(props);
-    this.state = {
-      events: [],
-    };
-  }
+  readonly state: AppState = {
+    events: [],
+    selectedChannel: null,
+  };
 
-  //TODO: change fetch API call to service
+  //TODO: what is being called to the App component here
+  //is still is up for debate. For example, currently I have all of the events
+  //from all of forever going to the calendar component at all times and then
+  //narrowing what I display in there.
   componentDidMount(): void {
     Promise.resolve(getEvents(DATA)).then(result =>
       this.setState({events: result}),
     );
   }
 
+  handleSelectChannel = (selected: number): void => {
+    this.setState({
+      selectedChannel: selected,
+    });
+  };
+
+  // TODO: widths for cells are very rough, will change! Also thinking about setting
+  // dimensions the same for all devices.
   render(): JSX.Element {
     return (
-      <div>
-        <Header title='AMP Release Calendar' />
-        <div>
-          <Calendar events={this.state.events} />
-        </div>
-      </div>
+      <Grid>
+        <Row>
+          <Cell desktopColumns={9} phoneColumns={3} tabletColumns={6}>
+            <Header title='AMP Release Calendar' />
+          </Cell>
+          <Cell
+            desktopColumns={2}
+            phoneColumns={1}
+            tabletColumns={2}
+            align={'middle'}>
+            TODO: Where the SpaceBar will go
+          </Cell>
+        </Row>
+        <hr></hr>
+        <Row>
+          <Cell desktopColumns={3} phoneColumns={1} tabletColumns={2}>
+            <span> TODO: Where RTVTable will go</span>
+            {/* TODO: currently only allows for single channel select, implement multiple selection with checkboxes */}
+            {/* TODO: figure out how to unhighlight a row after it is unselected */}
```

Comment:
This is a bug and should be fixed; not a TODO. Example:
![image](https://user-images.githubusercontent.com/44627152/77783649-9f809780-702f-11ea-811f-fc979fbae555.png)